src/main/resources/swagger/api-docs.yaml

```diff
@@ -975,6 +979,10 @@ paths:
           description: Snapshot ID of method.
           required: false
           type: number
+        - name: snapshotComment
+          in: query
+          type: string
+          description: Snapshot comment of AgoraEntity
         - name: synopsis
           in: query
           description: Synopsis of method.
```

Comment:
need to add snapshotComment to the `POST /api/methods/{namespace}/{name}/{snapshotId}` endpoint, around line 1192. That's the copy-snapshot endpoint.